src/unix.rs

```diff
@@ -64,21 +85,15 @@ mod tests {
 
     #[test]
     fn parse_identifier() {
-        let identifier = "fr_FR.UTF-8";
-        assert_eq!(parse_locale_code(identifier).as_deref(), Some(PARSE_LOCALE));
+        assert_eq!(posix_to_bcp47("fr_FR.UTF-8"), "fr-FR");
```

Comment:
I've added some constants.

Personally, I prefer using literals in tests, because they are more visual, but the version with constants is fine too.

For a large number of tests, however, I find using literals to be much more convenient.

Reply:
Its very situational, IMO. The nice things about constants is that they can be self-describing for people who haven't memorized everything about the input data.